src/components/dspPricing/PricingPlanOption.tsx

```diff
@@ -0,0 +1,172 @@
+        border: `1px solid ${theme.colors.grey400}`,
+        borderRadius: "12px",
+        display: "flex",
+        flex: 1,
+        flexDirection: "column",
+        opacity: active ? 1 : 0.5,
+        padding: 5,
+        position: "relative",
+      } }
+    >
+      <Box
+        sx={ {
+          backgroundColor: theme.colors.black,
+          border: `1px solid ${theme.colors.grey500}`,
+          borderRadius: "4px",
+          display: "flex",
+          justifyContent: "center",
+          left: `calc(50% - ${iconPxSize})`,
+          padding: "8px",
+          position: "absolute",
+          top: `-${iconPxSize}`,
+        } }
+      >
+        { iconElement }
+      </Box>
+
+      <Stack
+        sx={ {
+          alignItems: "center",
+          display: "flex",
+          gap: 3.75,
+          justifyContent: "center",
+        } }
+      >
+        <Stack textAlign="center">
+          <Typography variant="h4" sx={ { fontWeight: 700, pb: 2 } }>
+            { originalPricing && (
+              <Box
+                component="span"
+                sx={ { textDecoration: "line-through", mr: 1 } }
+              >
+                { originalPricing }
+              </Box>
+            ) }
+
+            { pricing === "Coming soon" ? (
+              <Box
+                component="span"
+                sx={ {
+                  background: theme.colors.lightGreen,
+                  borderRadius: "12px",
+                  color: theme.colors.green,
+                  fontSize: "14px",
+                  fontWeight: 500,
+                  px: 1.5,
+                  py: 0.5,
+                } }
+              >
+                { pricing }
+              </Box>
+            ) : (
+              pricing
+            ) }
+          </Typography>
+          <Typography variant="h2">{ title }</Typography>
+          <Typography
+            variant="subtitle1"
+            sx={ { fontWeight: 500, height: "60px" } }
+          >
+            { description }
+          </Typography>
+        </Stack>
+
+        <Stack gap={ 1.25 }>
+          { criteria.map((criterion, index) => (
+            <Stack
+              sx={ {
+                display: "flex",
+                flexDirection: "row",
+                gap: 1.5,
+              } }
+              key={ index }
+            >
+              { criterionIcon(criterion.includedInPlan) }
+              <Typography variant="body1" fontWeight={ 500 }>
+                {
+                  pricingPlanData.sharedCriterionText[
+                    criterion.criterionText as keyof typeof pricingPlanData.sharedCriterionText
+                  ]
+                }
+              </Typography>
+            </Stack>
+          )) }
+        </Stack>
+
+        <Divider sx={ { width: "70%" } } color={ theme.colors.grey400 } />
+
+        { buttonType === "primary" ? (
+          <Button
+            onClick={ () => {
+              dispatch(setIsArtistPricePlanSelected(true));
```

Comment:
I used redux to save the state for now, but this will need to be replaced by the backend endpoint once the server change has been merged.